apps/prairielearn/src/pages/editError/editError.html.ts

```diff
@@ -0,0 +1,102 @@
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+
+import { config } from '../../lib/config.js';
+import type { JobSequenceWithFormattedOutput } from '../../lib/server-jobs.js';
+
+export function EditError({
+  resLocals,
+  jobSequence,
+  failedSync,
+}: {
+  resLocals: any;
+  jobSequence: JobSequenceWithFormattedOutput;
+  failedSync: boolean;
+}) {
+  const { __csrf_token } = resLocals;
+
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${renderEjs(import.meta.url, "<%- include('../partials/head'); %>", {
+          ...resLocals,
+          pageTitle: 'Edit Failure',
+        })}
+      </head>
+
+      <body>
+        ${renderEjs(import.meta.url, "<%- include('../partials/navbar'); %>", {
+          ...resLocals,
+          navPage: 'error',
+        })}
+
+        <main id="content" class="container">
+          <script>
+            $(function () {
+              const button = document.querySelector('#results-some-uuid-button');
```

Comment:
I know this wasn't your change, but I'm confused why this is called `#results-some-uuid...`. Does this name make sense to you? I'd consider changing this.

Reply:
This likely comes from the instructor file editor page, which has the `results` button include a UUID, likely from copying code from the pl-file-editor. In #10233 I removed that UUID stuff (see https://github.com/PrairieLearn/PrairieLearn/pull/10233/commits/c592fcaedb95fcbb57a0eb0829768b52192816eb). I can do the same changes here as well for consistency.

Reply:
Renamed in 5191a74. Also took the opportunity to replace the click event with bootstrap events to clean it up.